index.html

```diff
@@ -112,8 +108,8 @@
       <p>
         A conforming <b>Fedora server</b> is a conforming [[!LDP]] 1.0 server except where described in this document
         that also follows the rules defined by Fedora in <a href="#resource-management"></a>,
-        <a href="#resource-versioning"></a>, <a href="#binary-fixity"></a>, <a href="#resource-authorization"></a>,
-        <a href="#atomic-operations"></a>, and <a href="#messaging"></a>.
+        <a href="#resource-versioning"></a>, <a href="#binary-fixity"></a>, <a href="#resource-authorization"></a>
+        , and <a href="#messaging"></a>.
```

Comment:
This comma trailing after the preceding line will result in space before the comma